src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure implements GetFigureArea {
+    private double side;
+
+    public void draw() {
+        System.out.println("Figure - Square: side " + this.side
+                + " ,area " + this.getArea() + " color "
+                + this.getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+    public Square(double side) {

```

Comment:
The constructor for `Square` should also initialize the color if it is inherited from the `Figure` class. Consider adding a parameter for color and setting it using a setter method or directly if accessible.